js/interface.js

```diff
@@ -640,6 +640,8 @@ var FlSlider = (function() {
   return FlSlider;
 })();
 
+var flSlider = new FlSlider(data);
```

Comment:
wouldn't this be removed again by ESLint when it runs?

if the variable is not used, why not simply change this to

```
new FlSlider(data);
```

@tonytlwu thoughts?

Reply:
Indeed. If this is caught by ESLint, then the dev shouldn't be submitting this. Either add a comment to ignore es-lint on this line, or just use `new FlSlider(data);`.

Reply:
new FlSlider(data); also would be considered as an error because of this rule https://github.com/Fliplet/eslint-config-client/blob/171629ef4e210cd11f03db2c5b857dde8e675c2c/index.js#L98. I will add a comment instead.

Reply:
@MaksymShokin let's remove the `no-new` ESLint condition from our config. I know that we rely on `new Obj()` on a few components because of old code mostly.